enable_logging_API.gs

```diff
@@ -0,0 +1,25 @@
+/**
+* Enable the logging API for cloud project
+* @param {Number} projectNumber of the GCP project
+* @return {Boolean} API is enabled or not
+*/
+function enableLogginApisPvt(projectNumber) {
```

Comment:
Do you ensure that the project is an active project. If i recall correctly, if the project is not active, this will throw an error

Reply:
Yes
This function is called only when the project is an active project